src/smokeTest/java/uk/gov/hmcts/futurehearings/pip/smoke/common/rest/RestClient.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.futurehearings.pip.smoke.common.rest;
+
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public class RestClient {
```

Comment:
Rename RestClient to RestClientTemplate for file and class names to be consistent with other test types e.g. functional and acceptance tests